satpy/etc/readers/fci_l1c_nc.yaml

```diff
@@ -6,109 +6,268 @@ reader:
     Reader for FCI L1c data in NetCDF4 format.
     Used to read Meteosat Third Generation (MTG) Flexible
     Combined Imager (FCI) L1c data.
-  status: Beta for full-disc FDHSI and HRFI, RSS not supported yet
+  status: Beta for full-disc FDHSI, HRFI, and African dissemination format. RSS not supported yet
   supports_fsspec: true
   reader: !!python/name:satpy.readers.yaml_reader.GEOVariableSegmentYAMLReader
-  sensors: [ fci ]
+  sensors: [fci]
 
 # Source: MTG FCI L1 Product User Guide [FCIL1PUG]
 # https://www.eumetsat.int/media/45923
+required_netcdf_variables: &required-variables
```

Comment:
apologies, one more comment about these anchors & aliases... i think these variables can be defined under `fci_l1c_fdhsi`-file type (where they are needed the first time) and use an anchor there and then use an alias elsewhere. so a separate variable defined here is not needed...